common/config/rush/pnpm-lock.yaml

```diff
@@ -163,24 +163,24 @@ packages:
     dev: false
```

Comment:
I see unrelated changes here, could you please revert changes and then rerun `rush update`?

Reply:
Yes, thanks, updated.